packages/utils/functions/axios/createAxiosInstance/index.ts

```diff
@@ -75,3 +103,34 @@ export const {
   requestInterceptorId: requestInterceptorIdForFiles,
   responseInterceptorId: responseInterceptorIdForFiles,
 } = createAxiosInstance({ file: true })
+
+export const refreshAccessToken = async (
```

Comment:
this little guy might fit better at `token/refreshAccessToken`, maybe we can add a TODO to move it later?

Reply:
The problem with that is that the function gets called inside the `createAxiosInstance` but at the same time the function needs access to the axios instance to call the refresh endpoint, so we would need to figure out that. I'll add the TODO tho so one day we handle it better.